src/meta_schedule/schedule_rule/schedule_rule.cc

```diff
@@ -251,6 +251,29 @@ Array<ScheduleRule> ScheduleRule::DefaultHexagon() {
   };
 }
 
+Array<ScheduleRule> ScheduleRule::DefaultMicro() {
+  return {
+      ScheduleRule::AutoInline(
```

Comment:
Is there any specific reason that `ApplyCustomRule` is not included as a rule here?

Reply:
No, there was not. I copied the rules from the Hexagon rules a while ago, and the `ApplyCustomRule` was not added then. I added the rule.

Reply:
Hi, after you add the `ApplyCustomRule` you may also consider to add other rules in the current Hexagon rules like `InlineConstantScalars`.

Reply:
@zxybazh I think this could be in a follow up PR since the PR has been open for a while. thanks!

Reply:
Thanks @zxybazh, I already added the rest of Hexagon rules except the  `ParallelizeVectorizeUnroll` which is not needed on the micro targets.